### PR TITLE
Switch to LuLPeg, update readme and rockspec

### PR DESCRIPTION
**Pull request by RunsFor**
We decided to switch to [lulpeg](https://github.com/tarantool/lulpeg) for now, since it is written in lua and appears to be more stable. Unlike `lpeg`, `lulpeg` is available in tarantool ecosystem.

Anyway, we may return to this later, when performance issues pops up.

### Tests:
`tarantool tests/runner.lua` gives this:

IMAGE 2019-12-12 16:29:28

### CODE REVIEW

**Review by filonenko-mikhail — APPROVED** [review]

**Review by olegrok — APPROVED** [review]

File: graphql/parse.lua [hunk 1]
```diff
@@ -1,4 +1,4 @@
-local lpeg = require 'lpeg'
+local lpeg = require 'lulpeg'
```
Comment (Totktonada) [inline graphql/parse.lua:1]: Please, see #3 on this.

File: README.md [hunk 2]
```diff
@@ -107,7 +107,8 @@ Running tests
 ---
 
 ```lua
-lua tests/runner.lua
+tarantoolctl rocks make -- optionally
+tarantool tests/runner.lua
```
Comment (Totktonada) [inline README.md]: Is it shell or lua? Code highlight and a comment is lua, but commands are for shell.

File: graphql/parse.lua (hunk 1 above)
Comment (RunsFor) [reply graphql/parse.lua:1]: Discussed online and decided to stick to lulpeg for now.
Updated PR description.

**Comment by Totktonada** [comment]
> We decided to switch to [lulpeg](https://github.com/tarantool/lulpeg) for now, since it is written in lua and appears to be more stable. Unlike `lpeg`, `lulpeg` is available in tarantool ecosystem.

It is great that the motivation of the change is explicit now, but I would like to see it right in the commit message: this way I'll able to see why things were changed w/o jumps between `git log` and GitHub PRs.

I would also make it explicit that `lpeg` segfaults in some environments rather then vague 'less stable'.

Please, update the commit message and proceed (or just proceed if you're disagree with me).

While we're here: will we create a merge commit for each merged PR (`Merge pull request` button) or will keep master's history flat (`Rebase and merge` button)? Let's decide and keep it consistent.

Personally I prefer flat history, but let's decide based on preferences of ones who want to contribute here.

**Review by Totktonada — APPROVED** [review]
LGTM.

**Comment by Totktonada** [comment]
BTW, I suggest to close #3 from the corresponsing commit message, because in fact your commit fixes the issue.

**Comment by RunsFor** [comment]
> will we create a merge commit for each merged PR (Merge pull request button) or will keep master's history flat (Rebase and merge button)? Let's decide and keep it consistent.

There is also a "Squash and merge" button. What about that? [Docs](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#squash-and-merge-your-pull-request-commits) says it uses fast-forward merge. Lets try to keep it flat, but not require it to be flat.

**Comment by Totktonada** [comment]
`Squash and merge` button is what that should never be used, IMHO. Commits should be atomic.

I know, it is possible to keep short-term branches (ones where a developer working on a bug or a feature) full of intermediate changes and them squash them at once. The only reason to do so it to refraining from force pushes. However it is completely okay to force-push a short-term branch. So there are no pros in this process, only cons: unability to review changes in its final state.

Aside of this, considering commits atomicity, if you squash commits before push, then you are unable to propose a patchset with more then one commit within one PR.

I strongly against squashing commits before push. This approach leads to a mess in many ways and this just should not be used. Or maybe should be used only for toy projects, where no review process exists and vcs history readability is not matter at all.

**Pull request merged**